actions/get_user.py

```diff
@@ -8,7 +8,7 @@
     'FindUserAction'
 ]
 
-
+#
```

Comment:
Should have two blank lines causing the pipeline to fail.  Recommend leaving as is.

Reply:
I didn't make any changes to that action.